tensorflow/lite/kernels/internal/reference/resize_bilinear.h

```diff
@@ -0,0 +1,199 @@
+#include "tensorflow/lite/kernels/internal/cppmath.h"
```

Comment:
One request would be to branch this from reference_ops.h so that it is clear exactly what (if any changes are made to the reference implementation).

```
git cp reference_ops.h resize_bilinear.h
```
and then remove all the extra code. The idea is to make it clear that the only operation was the code being copied, and nothing else.

Reply:
I can't seem to find the `git cp` command. I don't know how to accomplish what you are describing. How would you do it without using `git cp`? @advaitjain